src/QuartzNET-DynamoDB.Tests/Unit/ExponentialBackoffWithRandomVariationTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Quartz.DynamoDB.DataModel.Storage;
+using Xunit;
+
+namespace Quartz.DynamoDB.Tests.Unit
+{
+    public class ExponentialBackoffWithRandomVariationTests
+    {
+        /// <summary>
+        /// First attempt sleep duration should be between 1 and 2.
+        /// </summary>
+        [Fact]
+        public void CalculateWaitDurationFirstAttempt()
+        {
+            TimeSpan result = ExponentialBackoffWithRandomVariation.CalculateWaitDuration(1);
+            Assert.True(result >= TimeSpan.FromSeconds(1));
+            Assert.True(result <= TimeSpan.FromSeconds(2));
+        }
```

Comment:
I changed my mind on this today. The throughput throttling windows are per second, so the first retry might do it.